src/createRollupConfig.ts

```diff
@@ -50,6 +51,9 @@ export async function createRollupConfig(opts: TsdxOptions) {
     tsconfigJSON = fs.readJSONSync(resolveApp('tsconfig.json'));
   } catch (e) {}
 
+  const pathTransformer = (service: any): any =>
+    transformPaths(service.getProgram());
```

Comment:
Can these be more specific than `any`? `transformPaths` is written in TS, so I think it can be.

Also do you know if the second arg isn't necessary? It has an `exclude` option, not sure if we should pass `tsconfigJSON.exclude` into there or if it's read automatically (and the `exclude` option is just an override maybe?? idk the docs aren't comprehensive)

Reply:
oh, yea) `transformPaths` has types LanguageService and CustomTransformer - need to use,
to be fixed

Reply:
https://github.com/zerkalica/zerollup/issues/22